hw/top_earlgrey/top_earlgrey.core

```diff
@@ -26,6 +26,7 @@ filesets:
       - lowrisc:ip:kmac
       - lowrisc:ip:otbn
       - lowrisc:prim:ram_1p_adv
```

Comment:
You can remove this dependency, I think.

Reply:
yes this is not needed anymore - removed.